README.md

```diff
@@ -120,6 +120,14 @@ the required build dependencies and then build Ceph from source.
 
     (docker)# setup-ceph.sh
 
+If you need to limit the number of build jobs please do this the following
+way for example:
```

Comment:
sorry for not noticing this before, but I think the phrase needs some improvement:
```suggestion
If you need to limit the number of build jobs you can do the following:
```